### ISSUE TITLE
Cropping of NoInv Crafter for POBAX - set to first 27 rows

### ISSUE THREAD

**Issue by Sathya98** [taodav/pobax#39]
I was looking into running the No Inventory crafter environment for some playing around, and looked into the environment wrapper in pobax/envs/wrappers/pixel.py  - and noticed the following:

    @functools.partial(jax.jit, static_argnums=(0, 2))
    def get_obs(self, obs, normalize):
        if not normalize:
            obs *= 255
        assert len(obs.shape) == 4
        obs = obs[:,:27, :, :]
        return obs
    
    def observation_space(self, params):
        low, high = 0, 255
        if self.normalize:
            high = 1
        return spaces.Dict({
            "obs": spaces.Box(
                low=low,
                high=high,
                shape=(
                    27,
                    33,
                    3,
                ),
            )
        })

I ran the following code to initialize the env:

    import jax
    from pobax.envs import get_env

    num_envs = 4
    rng = jax.random.PRNGKey(0)
    env, env_params = get_env("craftax_pixels", rng, num_envs=num_envs)

    rngs = jax.random.split(rng, num_envs)
    obs, state = env.reset(rngs, env_params)

    print(obs.obs.shape)

and I see that the shape is (4, 27, 110, 3), and not (4, 27, 33, 3) as described in the paper. And I could not spot a resize operation to change the scaling from 10px to 3px as stated in any of the wrappers/files. How do we fix this? Was a resize missing?

Image: version in code
<img width="110" height="27" alt="Image" src="https://github.com/user-attachments/assets/286f0df9-1501-424f-bd6b-c6eb5f15cb06" />

Image: full obs version
<img width="110" height="130" alt="Image" src="https://github.com/user-attachments/assets/b3d0252b-5ea4-4188-b11e-37c3cc0c06fb" />
